media-gfx/openvdb/files/openvdb-4.0.2-use-gnuinstalldirs.patch

```diff
@@ -0,0 +1,83 @@
+diff -purN a/openvdb/CMakeLists.txt b/openvdb/CMakeLists.txt
+--- a/openvdb/CMakeLists.txt	2017-07-31 19:16:32.000000000 +0100
++++ b/openvdb/CMakeLists.txt	2017-08-15 12:09:53.088731456 +0100
+@@ -25,6 +25,7 @@
+ # LIABILITY FOR ALL CLAIMS REGARDLESS OF THEIR BASIS EXCEED US$250.00.
+ #
+ PROJECT ( OpenVDBCore )
++INCLUDE ( GNUInstallDirs )
+ 
+ IF ( USE_GLFW3 )
+   FIND_PACKAGE ( GLFW3 REQUIRED )
+@@ -401,7 +402,7 @@ IF (OPENVDB_BUILD_DOCS)
+     )
+ 
+   INSTALL ( CODE "EXECUTE_PROCESS(COMMAND \"${CMAKE_COMMAND}\" --build  \"${PROJECT_BINARY_DIR}\" --target doc)")
+-  INSTALL ( DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/doc/html DESTINATION docs)
++  INSTALL ( DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/doc/html DESTINATION ${CMAKE_INSTALL_DOCDIR})
+ 
+ ENDIF ()
+ 
+@@ -410,37 +411,37 @@ IF ( NOT WIN32 )
+   INSTALL ( TARGETS
+     vdb_view
+     DESTINATION
+-    bin
++    ${CMAKE_INSTALL_BINDIR}
+     )
+ ENDIF ()
+ INSTALL ( TARGETS
+   vdb_print
+   vdb_render
+   DESTINATION
+-  bin
++  ${CMAKE_INSTALL_BINDIR}
+   )
+ 
+ INSTALL ( TARGETS
+   openvdb_static
+   DESTINATION
+-  lib
++  ${CMAKE_INSTALL_LIBDIR}
+   )
+ 
+ IF (WIN32)
+   INSTALL ( TARGETS
+ 	openvdb_shared
+-	RUNTIME DESTINATION bin COMPONENT Runtime
+-	ARCHIVE DESTINATION lib COMPONENT Runtime
++	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT Runtime
++	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT Runtime
```

Comment:
afaik this is also missing a `LIBRARY` argument (not important, only a Windoze problem)